api/context/store/store.go

```diff
@@ -57,7 +57,7 @@ const (
 	LocalContextType = "local"
 	// KubeContextType is the endpoint key in the context endpoints for a new
 	// kube backend
-	KubeContextType = "kubernetes"
+	KubeContextType = "k8s"
```

Comment:
Is there really a conflict here ? if we know a context is of type kubernetes, the endpoint will also be named kubernetes, but won't be stored next to a "docker" endpoint ? (Happy to discuss this in more details)

Reply:
UX should use `kubernetes`, internal type can be `kube`, `k8s.io` or anything else